rules.adoc

```diff
@@ -740,7 +740,7 @@ robot’s dimensions must not exceed the following limits:
 
 |===
 |sub-league | *Soccer* *Open* | *Soccer Lightweight*
-|size / (robot must fit cylinder of this diameter) | 18.0 cm | 22.0 cm +
+|size / (robot must fit smoothly cylinder of this diameter) | 18.0 cm | 22.0 cm +
```

Comment:
```suggestion
|size / (robot must fit {++smoothly++} cylinder of this diameter) | 18.0 cm | 22.0 cm +
```

@aquahika this is a small change so that the change will end up being tracked in the list and gets bolded out in red :slightly_smiling_face: